tests/scripts/thread-cert/Cert_5_2_05_AddressQuery.py

```diff
@@ -30,110 +30,130 @@
 import time
 import unittest
 
+import config
+import network_layer
 import node
-
-ED1 = 1
-BR1 = 2
-LEADER = 3
-ROUTER2 = 4
-REED = 5
-ED2 = 6
-ED3 = 7
+import ipv6
+
+LEADER = 1
+ROUTER1 = 2
+BR = 3
+ED1 = 17
+DUT_REED = 18
+SNIFFER = 34
+ROUTER_SELECTION_JITTER = 1
+MESH_LOCAL_PREFIX = 'fdde:ad00:beef:0:'
+ROUTING_LACATOR = ':0:ff:fe00'
 
 class Cert_5_2_5_AddressQuery(unittest.TestCase):
+    def checkAddressNotification(self):
+        """Check if the DUT_REED sent a properly formatted Address Notification message or not.
+        """
+        reed_messages = self.sniffer.get_messages_sent_by(DUT_REED)
+        msg = reed_messages.next_coap_message('0.02')
+        msg.assertCoapMessageRequestUriPath('/a/an')
+        msg.assertCoapMessageContainsTlv(network_layer.TargetEid)
+        msg.assertCoapMessageContainsOptionalTlv(network_layer.Rloc16)
```

Comment:
How about moving `checkAddressNotification` to a common component so that it could be reusable by other tests? (as there are other tests in Test Plan would check Address Notification)

Reply:
1. fixed.
2.Yes. I will move it to a common component when I meet another test which needs it.